pkg/tagger/collectors/kubelet_main.go

```diff
@@ -40,6 +41,10 @@ type KubeletCollector struct {
 
 // Detect tries to connect to the kubelet
 func (c *KubeletCollector) Detect(out chan<- []*TagInfo) (CollectionMode, error) {
+	if !config.IsKubernetes() {
```

Comment:
Should we consolidate and use `config.IsFeaturePresent(config.Kubernetes)`?

Reply:
Yes `AUTCONFIG_FROM_ENVIRONMENT` can be set to `false`, nice catch! Let's keep using `!config.IsKubernetes()` then.